cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -2,7 +2,7 @@ name "AWS Idle Compute Instances"
 rs_pt_ver 20180301
 type "policy"
 short_description "Gathers AWS CloudWatch data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/idle_compute_instances/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
should reference it's for idle instance, like what is in the What it does of the readme